core/build.gradle

```diff
@@ -3,6 +3,7 @@ apply plugin: 'bintray-release'
 
 dependencies {
     compile 'com.android.support:support-v4:23.0.1'
+    compile 'io.reactivex:rxandroid:1.0.1'
```

Comment:
Yeah. It'll have to be included I believe. AFAIK you can't use `provided` scope for `aar` dependencies. You can only use it with `jar` dependencies.